Source/WebCore/platform/mac/ScrollbarThemeMac.mm

```diff
@@ -155,29 +155,19 @@ + (void)registerAsObserver
     if (scrollbar.isCustomScrollbar() || !scrollbar.shouldRegisterScrollbar())
         return;
 
-    bool isHorizontal = scrollbar.orientation() == ScrollbarOrientation::Horizontal;
-    auto scrollerImp = retainPtr([NSScrollerImp scrollerImpWithStyle:ScrollerStyle::recommendedScrollerStyle() controlSize:nsControlSizeFromScrollbarWidth(scrollbar.widthStyle()) horizontal:isHorizontal replacingScrollerImp:nil]);
-    scrollbarMap().add(&scrollbar, WTFMove(scrollerImp));
-    didCreateScrollerImp(scrollbar);
-    updateEnabledState(scrollbar);
-    updateScrollbarOverlayStyle(scrollbar);
+    scrollbarMap().add(&scrollbar);
 }
 
 void ScrollbarThemeMac::unregisterScrollbar(Scrollbar& scrollbar)
 {
-    [scrollbarMap().take(&scrollbar) setDelegate:nil];
-}
-
-void ScrollbarThemeMac::setNewPainterForScrollbar(Scrollbar& scrollbar, RetainPtr<NSScrollerImp>&& newPainter)
-{
-    scrollbarMap().set(&scrollbar, WTFMove(newPainter));
-    updateEnabledState(scrollbar);
-    updateScrollbarOverlayStyle(scrollbar);
+    scrollbarMap().take(&scrollbar);
 }
 
 NSScrollerImp *ScrollbarThemeMac::painterForScrollbar(Scrollbar& scrollbar)
```

Comment:
I wonder if it's time to retire the term "painter" and just say "scrollerImp".

Reply:
Changed to scrollerImp